archunit-junit/junit5/engine/src/testFixtures/java/com/example/SomeClass.java

```diff
@@ -0,0 +1,19 @@
+package com.example;
+
+import java.io.Serializable;
+
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.lang.ArchRule;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+
+public class SomeClass {
```

Comment:
Those files need to exist in order for `FieldSource.from` / `FieldSelector.selectField` to not throw `NoClassDefFoundError`s